package.json

```diff
@@ -29,6 +29,7 @@
     "@types/lodash": "^4.14.149",
```

Comment:
I thought lodash was killed? only in the server?

Reply:
yes, only in the server